src/aof.c

```diff
@@ -426,24 +429,24 @@ void markRewrittenIncrAofAsHistory(aofManifest *am) {
     listNode *ln;
     listIter li;
 
-    listRewindTail(am->incr_aof_list,&li);
+    listRewindTail(am->incr_aof_list, &li);
 
-    /* server.aof_fd != -1 means that AOF is open, then we must skip the last 
-     * AOF, because this one is our currently writing. */
+    /* server.aof_fd != -1 means that AOF is open, then we must s
+     * kip the last AOF, because this one is our currently writing. */
```

Comment:
```suggestion
    /* server.aof_fd != -1 means that AOF is open, then we must
     * skip the last AOF, because this one is our currently writing. */
```